utils/hwloc/hwloc-tbind.h

```diff
@@ -0,0 +1,314 @@
+/***************************************************************************
+ * Copyright 2019 UChicago Argonne, LLC.
+ * Author: Nicolas Denoyelle
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please put in the following line in your headers in new files (per other files in this repo):

```
 * See COPYING in top-level directory.
```

Reply:
Done see#https://github.com/open-mpi/hwloc/pull/359/commits/028c2eeaab0ba915743f068c85e5d4a8b0cfeaf5